src/content/developers/docs/intro-to-ethereum/index.md

```diff
@@ -43,7 +43,7 @@ Any participant who broadcasts a transaction request must also offer some amount
 
 The amount of ether paid is a function of the length of the computation. This also prevents malicious participants from intentionally clogging the network by requesting the execution of infinite loops or resource-intensive scripts, as these actors will be continually charged.
 
-## What are dapps? {#what-are-dapps}
+## What are smart contracts? {#what-are-smart-conracts}
```

Comment:
Sorry, I know I just approved but noticed the typo in the link name, mind fixing that first?

Reply:
yea oof, thanks for calling out the typo!